### DISCUSSION TITLE
How to shrink existing EPHEMERAL disk

### DISCUSSION THREAD

**Discussion by eshikhov** [siderolabs/talos#9373] [Q&A]
Hi!

Updated from talos 1.7.6 to 1.8.0, how to properly shrink an existing EPHEMERAL disk?

**Comment by smira** [comment]
There is no supported way at the moment so that Talos would shrink it.

If you start from a completely wiped system (without `EPHEMERAL`), you can apply volume config to make it smaller, but there's no migration story.

You could of course shrink it manually by using some live CD, but that's outside of the scope.
